aries-site/src/pages/components/avatar.mdx

```diff
@@ -0,0 +1,111 @@
+import { BestPracticeGroup, Example } from '../../layouts';
+import {
+  AvatarAccessibilityExample,
+  AvatarCardExampleDo,
+  AvatarCardExampleDont,
+  AvatarDataTableExampleDo,
+  AvatarDataTableExampleDont,
+  AvatarPageHeaderExample,
+  AvatarSizePreviewExample,
+} from '../../examples';
+
+## Anatomy
+
+Avatars can be displayed as an image, initials, or a logo. They come in three different sizes:
+
+- `small`: 24x24px
+- `medium`: 48x48px
+- `large`: 96x96px
+
+The container of an Avatar can be circular or a rounded square based on the use case.
+
+- Use a circle Avatar when representing a person either through an image or their initials.
+- Use a rounded square Avatar when representing a company’s or entity’s logo. This can be achieved by passing `round='medium'`
+- Always round the edges of a square container when using an avatar.
+
+<Example
+  figma="https://www.figma.com/file/d7a74LgBABdgTZR6HrzWKo/HPE-Avatar?node-id=0%3A1"
+  code="https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/components/avatar/AvatarSizePreviewExample.js"
+>
+  <AvatarSizePreviewExample />
+</Example>
+
+## Accessibility
+
+Inform the user of what the Avatar is and what will happen when it is clicked by including an `a11yTitle`.
+
+<Example
+  horizontalLayout
+  code="https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/components/avatar/AvatarAccessibilityExample.js"
+  width={{ min: 'medium' }}
+>
+  <AvatarAccessibilityExample />
+</Example>
+
+## When to use Avatars
+
+Typically, Avatars are used mainly as an easy identifier for users and are always paired with additional information for better clarity. If possible, prioritize using an image or logo over initials.
+
+## Variants
+
+### DataTable
+
+When using Avatars in a DataTable, always provide the user’s or company’s name or email alongside it. If an image is not available provide an Avatar with initials instead. Use small-sized avatars in DataTables.
+
+<BestPracticeGroup columns="100%">
+  <Example
+    height="none"
+    bestPractice={{
+      type: 'do',
+      message:
+        'Including Avatars here helps the user to quickly scan the table.',
+    }}
+  >
+    <AvatarDataTableExampleDo />
+  </Example>
+  <Example
+    height="none"
+    bestPractice={{
+      type: 'dont',
+      message: `Don't remove an avatar if one is not available. This breaks visual consistency. Instead use initials as a placeholder.`,
+    }}
+  >
+    <AvatarDataTableExampleDont />
+  </Example>
+</BestPracticeGroup>
+
+### Card
+
+Both Avatars and Icons can be used within Cards. Reminder that...
+
+- Avatars represent a person, company, or entity.
+- Icons represent objects, actions, programs, and other communicative symbols.
+
+Use medium-sized avatars within cards.
+
```

Comment:
Doesn't look like the Avatars in this example have medium rounding

Reply:
missed that Ill add

Reply:
The logo doesn't fill the container, so the rounding doesn't seem like it would apply. Is that the right way to think about it?

Reply:
@taysea It would affect it in dark mode, looks like there is a white background fill. Whether or not that should be there is a different question 
![image](https://user-images.githubusercontent.com/86075038/196746078-6807ccef-7a3d-42d3-bf99-d464fc0dda12.png)

Reply:
I think most company logos in general have dark mode versions but not sure if the Grommet icon library has those as well. Looks like the current solution is to include a white background, but not entirely sure if that was a conscious decision or an importing error.

Reply:
> I think most company logos in general have dark mode versions but not sure if the Grommet icon library has those as well.

Many logos don't exist in grommet icons library, and instead the image is uploaded and stored as an svg separately (that's what we have done in these examples). So if a dark mode version is available that could be added.

> Looks like the current solution is to include a white background, but not entirely sure if that was a conscious decision or an importing error.

When you say "conscious decision or an importing error" do you mean in the site or in Figma?

Reply:
What I'm getting at is trying to understand what the design desire is and what the color contrast considerations are. It feels like:
1. If a dark mode version is available, we should use that with no background
2. If there is no dark mode version available, we should have a treatment to ensure the logo is visible (white feels a bit stark to me, but that's just my opinion)

Reply:
Sorry, my previous comment was pretty confusing.

I agree that if it's available, a dark-mode version of a logo should always be used with no background. For logos with no dark mode version, I didn't know that the white background was changeable. Very open to explore different treatments here.

Reply:
>>When you say "conscious decision or an importing error" do you mean in the site or in Figma?

Oops, thought that the white background was something all logos had by default within the Grommet icons library.